libbeat/tests/system/input/libbeat.yml

```diff
@@ -106,7 +106,7 @@ processors:
 #logging.selectors: ["*"]
 
 #============================== Xpack Monitoring ===============================
-# beatname can export internal metrics to a central Elasticsearch monitoring
+# libbeat can export internal metrics to a central Elasticsearch monitoring
```

Comment:
Should this be a title too?

Reply:
It is just a test file, so I don't think that matters. :D

Reply:
:D